opm/input/eclipse/Schedule/GasLiftOpt.hpp

```diff
@@ -181,43 +311,165 @@ class GasLiftWell {
         serializer(m_alloc_extra_gas);
     }
 
+    /// Create a serialisation test object.
     static GasLiftWell serializationTestObject();
 
+    /// Equality predicate.
+    ///
+    /// \param[in] other Object against which \code *this \endcode will be
+    /// tested for equality.
+    ///
+    /// \return Whether or not \code *this \endcode is the same as \p
+    /// other.
     bool operator==(const GasLiftWell& other) const;
 
 private:
-    std::string m_name;
-    std::optional<double> m_max_rate;
-    double m_min_rate = 0;
-    bool m_use_glo = false;
-    double m_weight = 1;
-    double m_inc_weight = 0;
-    bool m_alloc_extra_gas = false;
+    /// Well name.
+    std::string m_name{};
+
+    /// Maximum lift gas injection limit.
+    ///
+    /// Nullopt value interpretation subject to value of m_use_glo.
+    std::optional<double> m_max_rate{};
+
+    /// Minimum lift gas injection rate.
+    double m_min_rate { 0.0 };
+
+    /// Whether or not gas lift optimisation applies to this well.
+    bool m_use_glo { false };
+
+    /// Weighting factor for preferential allocation of lift gas.
+    double m_weight { 1.0 };
+
+    /// Weighting factor for incremental gas rate.
+    double m_inc_weight { 0.0 };
+
+    /// Whether or not to allocate extra lift gas to this well even if group
+    /// target is or would be exceeded.
+    bool m_alloc_extra_gas { false };
 };
 
-class GasLiftOpt {
+// ---------------------------------------------------------------------------
+
+/// Gas lift optimisation parameters for all wells and groups.
+class GasLiftOpt
+{
 public:
+    /// Retrieve gas lift optimisation parameters for a single named group.
+    ///
+    /// Throws an exception of type std::out_of_range if no parameters have
+    /// been defined for the named group.
+    ///
+    /// \param[in] gname Group name.
+    ///
+    /// \return Gas lift optimisation parameters for named group \p gname.
     const GasLiftGroup& group(const std::string& gname) const;
+
+    /// Retrieve gas lift and gas lift optimisation parameters for a single
+    /// named well.
+    ///
+    /// Throws an exception of type std::out_of_range if no parameters have
+    /// been defined for the named well.
+    ///
+    /// \param[in] qname Well name.
+    ///
+    /// \return Gas lift and gas lift optimisation parameters for named
+    /// well \p wname.
     const GasLiftWell& well(const std::string& wname) const;
 
+    /// Lift gas rate increment.
     double gaslift_increment() const;
+
+    /// Assign lift gas rate increment.
+    ///
+    /// \param[in] gaslift_increment Lift gas rate increment value.
     void gaslift_increment(double gaslift_increment);
+
+    /// Retreive minimum economical gradient threshold to continue
```

Comment:
Retrieve

Reply:
> Retrieve

Good catch.  I've pushed an update to address this.